test/metabase/api/database_test.clj

```diff
@@ -223,20 +230,20 @@
 (defn- create-db-via-api! [& [m]]
   (let [db-name (mt/random-name)]
     (try
-      (let [{db-id :id, :as response} (with-redefs [driver/available?   (constantly true)
-                                                    driver/can-connect? (constantly true)]
-                                        (mt/user-http-request :crowberto :post 200 "database"
-                                                              (merge
-                                                               {:name    db-name
-                                                                :engine  (u/qualified-name ::test-driver)
-                                                                :details {:db "my_db"}}
-                                                               m)))]
-        (is (schema= {:id       s/Int
-                      s/Keyword s/Any}
-                     response))
-        (when (integer? db-id)
-          (t2/select-one Database :id db-id)))
-      (finally (t2/delete! Database :name db-name)))))
+     (let [{db-id :id, :as response} (with-redefs [driver/available?   (constantly true)
+                                                   driver/can-connect? (constantly true)]
+                                       (mt/user-http-request :crowberto :post 200 "database"
+                                                             (merge
+                                                              {:name    db-name
+                                                               :engine  (u/qualified-name ::test-driver)
+                                                               :details {:db "my_db"}}
+                                                              m)))]
+       (is (malli= [:and
+                    [:map-of :keyword :any]
+                    [:map [:id pos-int?]]]
+                   response))
```

Comment:
This is essentially equivalent. The schema test only had `s/Keyword` because it needs that to allow other things in the map.
```suggestion
       (is (malli= [:map [:id pos-int?]]]
                   response))
```